tpch/execute.py

```diff
@@ -40,7 +40,7 @@
     "dask": lambda x: x.compute(),
 }
 
-DUCKDB_SKIPS = ["q11", "q14", "q15", "q22"]
+DUCKDB_SKIPS = ["q14", "q15"]
```

Comment:
🙀 only 2 left!

unfortunately I don't think these are possible without them introducing `duckdb.WindowExpression` so they may need to stay parked for now